hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import styles from "./TeamDetail.module.scss";
+import Header from "components/general/Header/Header";
+import Typography from "@material-ui/core/Typography";
+import { mockTeamMultiple } from "testing/mockData";
+import {
+    Table,
+    TableBody,
+    TableContainer,
+    TableHead,
+    TableRow,
+    TableCell,
+    Checkbox,
+    Paper,
+    Container,
+    Box,
+} from "@material-ui/core";
+
+const TeamDetail = () => {
+    return (
+        <>
+            <Header />
```

Comment:
You did a really good job with css! But usually, we try to stick with MUI components as much as we can and write custom css sparingly.
Here, I think it'd be a good place to use MUI Grid instead of the container and table container you have here (https://v4.mui.com/components/grid/)
and use the size breakpoints to test how it looks on different screens.